build_projects/shared-build-targets-utils/Utils/HostVersion.cs

```diff
@@ -64,9 +64,9 @@ public override string ToString()
         public string LatestHostBuildMajor => CommitCountString;
         public string LatestHostBuildMinor => "00";
         public bool EnsureStableVersion => false;
-        public VerInfo LatestHostVersion => new VerInfo(1, 0, 2, ReleaseSuffix, LatestHostBuildMajor, LatestHostBuildMinor, CommitCountString);
-        public VerInfo LatestHostFxrVersion => new VerInfo(1, 0, 2, ReleaseSuffix, LatestHostBuildMajor, LatestHostBuildMinor, CommitCountString);
-        public VerInfo LatestHostPolicyVersion => new VerInfo(1, 0, 2, ReleaseSuffix, LatestHostBuildMajor, LatestHostBuildMinor, CommitCountString);
+        public VerInfo LatestHostVersion => new VerInfo(1, 2, 0, ReleaseSuffix, LatestHostBuildMajor, LatestHostBuildMinor, CommitCountString);
```

Comment:
just curious why are the major and minor versions hard coded while the others are parameterized ?